packages/core/src/login/oidc/oidcHandlers/AuthorizationCodeWithPkceOidcHandler.ts

```diff
@@ -94,7 +104,11 @@ export default abstract class AuthorizationCodeWithPkceOidcHandlerBase {
         issuer: oidcLoginOptions.issuer.toString(),
         // The redirect URL is read after redirect, so it must be stored now.
         redirectUrl: oidcLoginOptions.redirectUrl,
-        dpop: oidcLoginOptions.dpop ? "true" : "false",
+        dpop: Boolean(oidcLoginOptions.dpop).toString(),
+        keepAlive: booleanWithFallback(
+          oidcLoginOptions.keepAlive,
+          true,
```

Comment:
The default `true` seems to be set in multiple places. Its also set [here](https://github.com/inrupt/solid-client-authn-js/pull/3473/files#diff-659ac87dfd3711f4cfcea3c7bf6970980f4740fd59df45f04c7977bffaa23e98R86), [here](https://github.com/inrupt/solid-client-authn-js/pull/3473/files#diff-457298190cdad0e611f3ced3efa5d7dbcc9c199d3425ef0803ecfa1fc0c7216bR164) and [here](https://github.com/inrupt/solid-client-authn-js/pull/3473/files#diff-12a917a86414ec22d88e79dbb0dbc0c7a1f578d55c0686b36a582eb78fe9e083R183). Consider having it set in one place.

Reply:
This can be left as is for now and it can be a future tidy up.